_dprhtml/features/bottom-pane/init.js

```diff
@@ -1,6 +1,14 @@
 'use strict';
 
 class BottomPaneTabsViewModel {
+  static TabNumber = {
```

Comment:
Does this seem like a reasonable place for this enum to live?

Reply:
Since we discussed the DPR1_chrome_mod API should be used instead, I'm going to remove this enum for now since it isn't being used. We can always reintroduce if we touch code that uses the lower-level showBottomPane API.